sdk/test-utils/test-utils/src/index.ts

```diff
@@ -15,3 +15,4 @@ export { isNode, isNode8 } from "./utils";
 export { TestSpan } from "./tracing/testSpan";
 export * from "./tracing/testTracer";
 export * from "./tracing/testTracerProvider";
+export { trace } from "@opentelemetry/api";
```

Comment:
I wanted to call this out... right now I am just exporting this as `trace` but that's a really generic name and not very meaningful unless you're pulling it from `@opentelemetry/api`...

Options are:
1. Rename it to something more meaningful like openTelemetryTrace
2. Bundle everything under the openTelemetry namespace and just export that
3. Keep it as trace
4. Export the individual wrapper methods like we do in core-tracing today

The benefit of (3) is that it'll be easily findable but the downside of course is that `trace` is not a great name

import { trace } from "@azure/test-utils"... 

before I go and make a ton of changes I figured I'll bring this up and see what folks think...

Reply:
I'm leaning towards either 1 or 2 depending on if we think there will be other things we want to re-export from OT